src/configs/character.rs

```diff
@@ -19,6 +22,9 @@ impl<'a> Default for CharacterConfig<'a> {
             success_symbol: "[❯](bold green)",
             error_symbol: "[❯](bold red)",
             vicmd_symbol: "[❮](bold green)",
+            root_success_symbol: "[#](bold green)",
+            root_error_symbol: "[#](bold red)",
+            root_symbol_enabled: false,
```

Comment:
I'm a little concerned by the amount of added configuration options something like this adds.

Ideally, we would have `success_style`, `error_style`, `symbol` and `root_symbol`; similarly to how we have `style`, `not_capable_style`, and `symbol` for the Node.js module. The trade-off there being that folks wouldn't be able to change the error symbol. 🤔

I'd be curious to hear the thoughts of other @starship/astronauts about how we could go about handling these cases.